app/controllers/idv/socure/document_capture_controller.rb

```diff
@@ -56,7 +56,24 @@ def show
       end
 
       def update
-        render plain: 'stub to ensure Socure callback exists and the route works'
+        clear_future_steps!
+        idv_session.redo_document_capture = nil # done with this redo
+        # Not used in standard flow, here for data consistency with hybrid flow.
+        document_capture_session.confirm_ocr
+
+        result = handle_stored_result
+        # TODO: new analytics event?
+        analytics.idv_doc_auth_document_capture_submitted(**result.to_h.merge(analytics_arguments))
+
+        Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
+          call('socure_document_capture', :update, true)
+
+        # cancel_establishing_in_person_enrollments
```

Comment:
is this a TODO?

Reply:
Right... copied over from non-socure. This is from a [bug fix](https://github.com/18F/identity-idp/pull/8712) - we should test it in my sandbox to see if it's needed.

Reply:
Implemented in 078a91ba91